daemon/reload.go

```diff
@@ -61,10 +64,7 @@ func (daemon *Daemon) Reload(conf *config.Config) (err error) {
 	if err := daemon.reloadLiveRestore(conf, attributes); err != nil {
 		return err
 	}
-	if err := daemon.reloadNetworkDiagnosticPort(conf, attributes); err != nil {
-		return err
-	}
-	return nil
+	return daemon.reloadNetworkDiagnosticPort(conf, attributes)
```

Comment:
I would not change this, otherwise if later you'll want to add another call, say `daemon.reloadFooBar`, you have to revert this back. Just a nipick of course.